study-builder/studies/rgp/i18n/es.conf

```diff
@@ -17,15 +17,330 @@
+  protect the privacy of health information obtained for research. This is an abbreviated notice, and does not
+  describe all details of this requirement (see Partners Privacy Notice*). During this study, identifiable
+  information about you or your health will be collected and shared with the researchers conducting the research.
+  In general, under federal law, identifiable health information is private. However, there are exceptions to
+  this rule. In some cases, others may see your identifiable health information for purposes of research oversight,
+  quality control, public health and safety, or law enforcement. We share your health information only when we
+  must, and we ask anyone who receives it from us to protect your privacy.""",
+  "enrollment_hipaa_p2": "[es]*Partners HealthCare Notice for Use and Sharing of Protected Health Information",
+  "enrollment_hipaa_p3": "http://www.partners.org/Assets/Documents/Notices/Partners_Privacy_Policy_English.pdf",
   "enrollment_p1_heading_a": "[es]Please provide information about yourself ",
-  "enrollment_p1_heading_b": "(person filling out this form):",
-  "enrollment_title_prompt": "[es]Your Title",
-  "enrollment_title_label": "[es]Title",
+  "enrollment_p1_heading_b": "[es](person filling out this form):",
+  "enrollment_title_label": "[es]Your Title *",
   "enrollment_title_mr": "[es]Mr.",
   "enrollment_title_mrs": "[es]Mrs.",
   "enrollment_title_miss": "[es]Miss",
   "enrollment_title_ms": "[es]Ms.",
   "enrollment_title_dr": "[es]Dr.",
-  "enrollment_title_required": "[es]Please indicate your title"
+  "enrollment_title_required": "[es]Please indicate your title",
+  "first_name_placeholder": "[es]Your First Name *",
+  "first_name_required": "[es]Please indicate your first name",
+  "phone_number_placeholder": "[es]Your telephone number *",
+  "phone_number_req_hint": "[es]Please enter a valid phone number with exactly and only 10 digits",
+  "phone_number_confirm": "[es]Confirm Your Telephone Number *",
+  "phone_number_mismatch": "[es]Please check your answer, phone numbers should match",
+  "relationship_label": "[es]The patient is [select from list] *",
+  "relationship_myself": "[es]Myself",
+  "relationship_minor": "[es]My minor (under age 18y) child",
+  "relationship_adult": "[es]My adult (age 18y or older) child who is unable to provide consent",
+  "relationship_other": "[es]Other",
+  "relationship_other_details_label": "[es]Please specify your relationship to the patient",
+  "relationship_req_hint": "[es]Please indicate your relationship to the patient",
+  "relationship_text": """[es]If the patient is age 18y or older, applications must be submitted by the patient themselves
+  (if they are able) or an individual with the authorization to provide medical information on their behalf.
+  Please call us at 617-714-7395 if you have questions.""",
+  "state_label": "[es]State the family lives in *",
+  "state_al": "[es]Alabama",
```

Comment:
Kind of curious what the Spanish names of the states are. Aren't they just the English names?

Reply:
Good point: they must be the English names.  I think I used find and replace to add the prefixes...